client/src/main/java/com/arcbees/beeshop/client/application/widget/sidepanel/cart/ShoppingCartView.java

```diff
@@ -54,7 +57,7 @@
 
     private void bind() {
         $(numberOfItemsTooltip).hide();
-        $(checkoutButton).hide();
+        $(checkoutContainer).hide();
 
         $(checkoutButton).click(new Function() {
```

Comment:
The button is disabled so it has no effect. I left it so it won't give warning for unused methods, but I can remove it if you think it's better.